docs/cloud/how-to-manage-certificates-in-temporal-cloud.md

```diff
@@ -31,31 +44,27 @@ An end-entity certificate _must_ meet the following criteria:
 
 When a client presents an end-entity certificate, and the whole certificate chain is constructed, each certificate in the chain (from end-entity to the root) must have a unique Distinguished Name (DN).
```

Comment:
might need to link to documentation on what DN means? Example: https://knowledge.digicert.com/generalinformation/INFO1745.html (if we can find a better document or codify that ourselves, that's probably better)

Reply:
The exhaustive definition is in section 9.5 of [ITU-T X.501](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-X.501-201910-I!!PDF-E&type=items), but it's not exactly light reading. Trying to boil it down to the level we need is tricky. I'll keep hunting for a more accessible definition (to which we can link, if from a reputable source, or that I can use as inspiration for an inline definition).